Reduce CIS benchmark documentation page contents (#17108)
fleetdm/fleet: ee/cis/macos-14/README.md

Reviewer: CIS_Apple_macOS_14.0_Sonoma_v1.0.0.pdf
https://drive.google.com/file/d/10hSWBZ06_XTwxC0GNjsjwObcQk72NQi8/view?usp=drive_link

# macOS 14.0 Sonoma benchmark

> Fleet's policies have been written against vTODO of the benchmark. You can refer to the [CIS website](https://www.cisecurity.org/cis-benchmarks) for full details about this version.

Reviewer 2: ```suggestion
Fleet's policies have been written against v1.0.0 of the benchmark. You can refer to the [CIS website](https://www.cisecurity.org/cis-benchmarks) for full details about this version.
```

We cover 1.0.0. @sharon-fdm please correct me if I'm wrong.

Here's where we're tracking updates to CIS Benchmark versions: https://docs.google.com/spreadsheets/d/1IWfQtSkOQgm_JIQZ0i2y3A8aaK5vQW1ayWRk6-4FOp0/edit#gid=0

Reviewer: @noahtalerman 

[Versions](https://docs.google.com/spreadsheets/d/1HZn7e4zpd5nfxY0MhSdX8MkGpdpxyLdSZjk0Auo1WaQ/edit#gid=1561077273)

macos 13 [1.0.0](https://drive.google.com/file/d/1VL8e2wmApkayv-9gM8zlyYljP6phsQhO/view?usp=share_link)
macos 14[1.0.0](https://drive.google.com/file/d/10hSWBZ06_XTwxC0GNjsjwObcQk72NQi8/view)
Win10 [2.0.0](https://drive.google.com/file/d/1agln3JBRkwJcefHibXzLMai31UuvTsDm/view)
Win11 [2.0.0](https://drive.google.com/file/d/1Ol_TKqB96M4R4QCG7vkwCbWVV6fpZZnK/view?usp=drive_link)

> These policies cover TODO of TODO CIS items that are able to be automated. For requirements and usage details, see the [CIS Benchmarks](https://fleetdm.com/docs/using-fleet/cis-benchmarks) documentation.

Reviewer 2: I'm not sure how many CIS Benchmarks Fleet covers for macOS 14. 

Hey @sharon-fdm do you know?

Reviewer: macos 13 [1.0.0](https://drive.google.com/file/d/1VL8e2wmApkayv-9gM8zlyYljP6phsQhO/view?usp=share_link)
macos 14[1.0.0](https://drive.google.com/file/d/10hSWBZ06_XTwxC0GNjsjwObcQk72NQi8/view)
Win10 [2.0.0](https://drive.google.com/file/d/1agln3JBRkwJcefHibXzLMai31UuvTsDm/view)
Win11 [2.0.0](https://drive.google.com/file/d/1Ol_TKqB96M4R4QCG7vkwCbWVV6fpZZnK/view?usp=drive_link)

Reviewer 2: How many for macOS 14? 

> These policies cover TODO of TODO CIS items that are able to be automated.

These are the TODOs in the macOS 14 CIS Benchmark docs @rachaelshaw and I are trying to wrap up.

Reviewer: Sorry @noahtalerman I misled you.
There are 109 policies for macos14.

(I previously counted it wrong

# macOS 14.0 Sonoma benchmark

Fleet's policies have been written against v1.0 of the benchmark. You can refer to the [CIS website](https://www.cisecurity.org/cis-benchmarks) for full details about this version.

For requirements and usage details, see the [CIS Benchmarks](https://fleetdm.com/docs/using-fleet/cis-benchmarks) documentation.

### Limitations

> The following CIS benchmark checks cannot be automated and must be addressed manually:

Reviewer 2: ```suggestion
The following CIS benchmark checks cannot be checked with a policy in Fleet:
```
